tooling/ssa_fuzzer/fuzzer/README.md

```diff
@@ -0,0 +1,17 @@
+## setup
```

Comment:
I would appreciate a proper readme with explanation of what the tool is doing, why we need to delete the rus toolchain, etc